### PR TITLE
Keras tuner

### PR DESCRIPTION
**Pull request by jakubczakon**

### CODE REVIEW

File: neptunecontrib/monitoring/kerastuner.py [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+    def exit(self):
+        pass
+
+
+def log_tuner_info(tuner, experiment=None, log_project_dir=True):
+    """Logs runs results and parameters to neptune.
+
+    Logs all hyperparameter optimization results to Neptune. Those include best score ('best_score' metric),
+    best parameters ('best_parameters' property), score for every run ('run_score', metric),
```
Comment (PiotrJander) [inline neptunecontrib/monitoring/kerastuner.py:85]: `'run_score', metric` -> `'run_score' metric`

File: neptunecontrib/monitoring/kerastuner.py [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+    tuner project directory as artifact, ('hyperparameters/space' text log), tuner id ('tuner_id' property),
```
Comment (PiotrJander) [inline neptunecontrib/monitoring/kerastuner.py:86]: `as artifact` -> `as an artifact`

Comment (PiotrJander) [reply neptunecontrib/monitoring/kerastuner.py:86]: `, ('hyperparameters/space' text log),` ?

**Review by PiotrJander — APPROVED** [review]
I haven't run the code (let me know if I should) but otherwise LGTM (except a few apparent minor typos)
